### ISSUE TITLE
Dark mode no longer working independently of taskbar after Windows Update 

### ISSUE THREAD

**Issue by SupDos** [krlvm/BeautySearch#25]
![image](https://user-images.githubusercontent.com/6813986/164006264-dfe01925-8185-42f0-8592-54cdb35ed9d1.png)

After installing the April 12 KB5012599 update, the search UI only shows up in light mode, no matter what theme is picked in the BeautySearch installer

![explorer_2022-04-19_14-39-53](https://user-images.githubusercontent.com/6813986/164007211-355a67fb-b405-4da1-bca7-2adc5f1d6a28.png)

Win10 20H2, 19042.1645

Changing this setting makes it show in dark, while previously it showed in dark no matter what 
![image](https://user-images.githubusercontent.com/6813986/164007532-21aba4b3-31c0-4c9e-816e-40c04e5cf10a.png)

**Comment by krlvm** [comment]
>  search UI only shows up in light mode

How does it look?

**Comment by krlvm** [comment]
You can disable Enhanced Acrylic Design to achieve [this](https://raw.githubusercontent.com/krlvm/BeautySearch/master/github-images/tweaks/dark-theme/accent.png) look.

If you want to make it completely dark, open `C:\Windows\SystemApps\Microsoft.Windows.Search_cw5n1h2txyewy\cache\Local\Desktop\BeautySearch.js` in any text editor and add this code to the end of the file:
```js
injectStyle(`
    .${DEF_DARK} {
        background: #1f1f1f !important;
    }
`);
```

Then restart Search App, for example, by clicking the circle arrow ("Refresh") button in BeautySearch Installer.

**Comment by SupDos** [comment]
Thank you, that worked! Though for a brief moment it still shows the accent color while I assume the full search menu is loaded in

https://user-images.githubusercontent.com/6813986/164090728-cb44735b-e6ec-42cc-bfb3-913b1ccf0314.mp4

I'm assuming that this isn't something that can be fixed since that isn't part of what is being injected by BeautySearch? If so I will just have to learn to live with this. Feel free to close the issue if there's nothing that can be done, thank you for helping!

**Comment by krlvm** [comment]
Yes, unfortunately this is shown before the WebView is rendered
I will add this option as an optional tweak in the next release

**Comment by krlvm** [comment]
Available in [v1.14](https://github.com/krlvm/BeautySearch/releases/tag/v1.14)
